### PR TITLE
Update application properties file to include new parameter added

### PR DESCRIPTION
**Pull request by tynanford**
Reference is here: https://github.com/ChannelFinder/ChannelFinder-SpringBoot/issues/25

### CODE REVIEW

File: site-template/application.properties.in [hunk 1]
```diff
@@ -70,3 +70,6 @@ elasticsearch.property.type = cf_property
 
 elasticsearch.channel.index = channelfinder
 elasticsearch.channel.type = cf_channel
+
+# maximum query result size
+elasticsearch.query.size = 10000
```
Comment (jeonghanlee) [inline site-template/application.properties.in:75]: Is it possible to make the query.size configurable through `CONFIG_COMMOM`?

Comment (tynanford) [reply site-template/application.properties.in:75]: oh yes duh. I should have thought of that. I will make those changes. Currently fighting with ES and CF when trying to increase this size

Comment (tynanford) [reply site-template/application.properties.in]: added new commit with this feature

**Comment by jeonghanlee** [comment]
We will ignore the linter for bash at this moment. I will figure out to clean up later. Thank you, @tynanford .

**Pull request merged**